### PR TITLE
Fix I18n

### PR DESCRIPTION
**Pull request by BaloPhilip**
If remove any locales from langList config I18n,  the application still displayed all locales which are in  folder 'locales'

### CODE REVIEW

File: modules/look/client-react/ui-antd/components/LanguagePicker.jsx [hunk 1]
```diff
@@ -12,12 +12,19 @@ export default class LanguagePicker extends React.Component {
...
+    const languages = Object.keys(i18n.store.data);
+
     return (
-      <Dropdown overlay={this.getMenuItems()}>
-        <a className="ant-dropdown-link" href="#">
-          {this.props.i18n.language.slice(0, 2).toUpperCase()} <Icon type="down" />
-        </a>
-      </Dropdown>
+      <React.Fragment>
+        {languages.length > 1 && (
```
Comment (vlasenko) [inline modules/look/client-react/ui-antd/components/LanguagePicker.jsx]: What if languages.length === 1 ?

Comment (vlasenko) [reply modules/look/client-react/ui-antd/components/LanguagePicker.jsx]: Ah, nevermind, then we don't need to choose :)

**Comment by vlasenko** [comment]
LGTM, thank you!

File: modules/look/client-react/ui-antd/components/LanguagePicker.jsx (hunk 1 above)
Comment (evgenijolshanskij) [reply modules/look/client-react/ui-antd/components/LanguagePicker.jsx]: @vlasenko, In this case, we don't need the language picker to be displayed.

Comment (vlasenko) [reply modules/look/client-react/ui-antd/components/LanguagePicker.jsx]: @evgenijolshanskij Yes, exactly!

**Pull request merged**